### DISCUSSION TITLE
Retrieving Retry Schedule

### DISCUSSION THREAD

**Discussion by ryanwjackson** [sidekiq/sidekiq#5210] [Ideas]
We have an ask from users to be able to see the retry schedule for certain jobs. Is it possible to generate this schedule from the library? I saw [this issue](https://github.com/mperham/sidekiq/issues/2168) about how to compute it, but it would be better to have a source of truth rather than hard-coded numbers. 

One stop gap we have considered is to get the "next retry," but ideally we could show the schedule, what we have done so far, what's next, etc. for a specific job.

I did some digging, but couldn't find it. Is this provided in the lib somewhere?

**Comment by mperham** [comment]
Nope, there's no magic just this method which hasn't changed in years. It should be quite stable.

https://github.com/mperham/sidekiq/blob/091a77f4b0136faf4e230620056a2b65c017029c/lib/sidekiq/job_retry.rb#L218
